### PR TITLE
Adding  Camilo's bio to people.yml

### PR DESCRIPTION
**Pull request by kmilo9999**
Just a brief description on who I am.

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -183,7 +183,9 @@ data:
     title: Graphics/Visualization Software Engineer
     github_username: kmilo9999
     brown_directory_uuid: 448fac67-1dea-42ef-8205-9305ac9b79a6
-    bio: ""
+    bio: Camilo is a software engineer specialized in real time simulation engines, 3D modeling and rendering, animations and VR technologies.
```
Comment (eldu) [inline about/people.yml:186]:
```suggestion
    bio: Camilo is a software engineer who specializes in realtime simulation engines, 3D modeling and rendering, animations and VR technologies. 
```

File: about/people.yml [hunk 2]
```diff
@@ -183,7 +183,9 @@ data:
     title: Graphics/Visualization Software Engineer
     github_username: kmilo9999
     brown_directory_uuid: 448fac67-1dea-42ef-8205-9305ac9b79a6
-    bio: ""
+    bio: Camilo is a software engineer specialized in real time simulation engines, 3D modeling and rendering, animations and VR technologies. 
+      He loves learning programing languages, and enjoys developing applications in multiple frameworks. Outside of work, he passes his time developing his own video-games,
```
Comment (eldu) [inline about/people.yml:187]:
```suggestion
      He loves learning programing languages and enjoys developing applications in multiple frameworks. Outside of work, he passes his time developing his own video games, 
```

File: about/people.yml [hunk 3]
```diff
@@ -183,7 +183,9 @@ data:
     title: Graphics/Visualization Software Engineer
     github_username: kmilo9999
     brown_directory_uuid: 448fac67-1dea-42ef-8205-9305ac9b79a6
-    bio: ""
+    bio: Camilo is a software engineer specialized in real time simulation engines, 3D modeling and rendering, animations and VR technologies. 
+      He loves learning programing languages, and enjoys developing applications in multiple frameworks. Outside of work, he passes his time developing his own video-games, 
+      playing soccer and volunteering for multiple events accross Rhode Island.
```
Comment (eldu) [inline about/people.yml:188]:
```suggestion
      playing soccer and volunteering for multiple events across Rhode Island. 
```

**Review by eldu — CHANGES_REQUESTED** [review]
Just some spelling and punctuation updates ! Looks good !

**Comment by kmilo9999** [comment]
@eldu 
Applied the suggestions in your review.  Thanks you checking this PR.

**Review by ashleysyg — APPROVED** [review]

**Review by eldu — APPROVED** [review]
looks great !

**Pull request merged**